packages/database/lib/migrations/20250815160900_create_getting_started_meta_table.cjs

```diff
@@ -0,0 +1,23 @@
+exports.config = { transaction: false };
+
+/**
+ * @param {import('knex').Knex} knex
+ */
+exports.up = async function (knex) {
+    await knex.schema.createTable('getting_started_meta', (table) => {
+        table.increments('id').primary();
```

Comment:
it is recommended to use Identity column in recent pg. https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial